tests/v2/actions/clusters/clusterconfig.go

```diff
@@ -54,6 +55,7 @@ func ConvertConfigToClusterConfig(provisioningConfig *provisioningInput.Config)
 	newConfig.Registries = provisioningConfig.Registries
 	newConfig.UpgradeStrategy = provisioningConfig.UpgradeStrategy
 	newConfig.CloudProvider = provisioningConfig.CloudProvider
+	newConfig.EnableNetworkPolicy = provisioningConfig.EnableNetworkPolicy
```

Comment:
Does this need to be added to the Readme to explain the new setting, or is it okay to change it from the default in the new_cluster_existing_gitrepo test?

Reply:
personally I don't find it needed to add to the readme, and the default value (if it is nil) from rancher is false. But if you guys want it in the readme I can add it.

Reply:
It's no big deal to me. I don't like settings where I'm not sure what will happen if I set it to true, but since this is only used in one test and turned off by default, it doesn't matter.